stacks/azure/kustomization.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  - ../../admission-webhook/webhook/v3
+  - ../../common/centraldashboard/overlays/stacks
+  - ../../kubeflow-roles/base
+  - ./application/jupyter-web-app
+  - ../../jupyter/notebook-controller/base_v3
+  - ../../profiles/base_v3
+  - ./application/spark-operator
+  - ./application/spartakus
+  - ./application/tensorboard
```

Comment:
Likewise, this should be removed.

Reply:
I actually already had a commit to remove this in my local branch, but hadn't pushed it. That's now done!